bitnami/postgresql-repmgr/12/debian-11/rootfs/opt/bitnami/scripts/librepmgr.sh

```diff
@@ -237,6 +249,56 @@ repmgr_get_primary_node() {
     echo "$primary_port"
 }
 
+
+
+########################
+# switchover standby after long primary wait
+# Globals:
+#   REPMGR_*
+#   POSTGRESQL_*
+# Arguments:
+#   None
+# Returns:
+#   None
+#########################
+
+repmgr_switchover_standby() {
+    local -i timeout="${REPMGR_WAIT_PRIMARY_TIME:-60}"
+
+    primary_host=""
```

Comment:
What is this variable used for? It is never assigned a value and referenced in the conditional at line 277.

Reply:
that variable it is not used any more, so i will delete that line